pilot/pkg/proxy/envoy/v1/testdata/authn-hello-mtls-off.yaml.golden

```diff
@@ -0,0 +1,6 @@
+destinations:
+- name: hello
+  port:
+    name: "http"
```

Comment:
Let’s not call it destinations as it conflicts with destination rules (cc @louiscryan / @zackbutcher). Sorry I failed to comment on the design doc. But we really really need to call this something else. And introduce notion of subsets probably?

Reply:
I'd like to align this with destination/destination rule concept. As a mater of fact, I use the same Destination proto for this, which have notion of subset.